renderdoc/driver/gl/gl_common.h

```diff
@@ -56,11 +56,17 @@ struct GLWindowingData
 };
 
 #elif ENABLED(RDOC_LINUX)
+
+#ifdef RENDERDOC_SUPPORT_GL
```

Comment:
OK, that's fair enough. You're right that it's not worth tackling right now if there are more complex conflicts.